Makefile

```diff
@@ -9,6 +9,9 @@ clean:
 doc:
 	dune build @doc --profile=release
 
+test:
+	dune runtest
```

Comment:
it depends a bit one where it's going to be invoked from, but you might want `--profile release` here

Reply:
It's meant for CI so I'd rather leave it as it is.